demos/default/pages/api/og.tsx

```diff
@@ -0,0 +1,48 @@
+import { ImageResponse } from '@vercel/og'
+import { NextRequest } from 'next/server'
+
+export const config = {
+  runtime: 'experimental-edge',
+}
+
+export default async function handler(req: NextRequest) {
+  const { searchParams } = req.nextUrl
+  const username = searchParams.get('username')
+
+  return new ImageResponse(
```

Comment:
For context where this component is coming from, see the Vercel example, https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation#getting-started